internal/server/metrics.go

```diff
@@ -31,14 +31,17 @@ type MetricsOpts struct {
 }
 
 type metrics struct {
-	registry *prometheus.Registry
-	counter  *prometheus.CounterVec
-	duration *prometheus.HistogramVec
+	registry      *prometheus.Registry
+	counter       *prometheus.CounterVec
+	duration      *prometheus.HistogramVec
+	UptimeCounter *prometheus.Counter
 }
```

Comment:
Inconsistent use of upper/lower case

Reply:
I think they should all starte with an upper case letter

Reply:
I forgot the Counter here, it's not being used, so I'm just dropping it.